### DISCUSSION TITLE
Confusion on some features (Profiles, containers, workspaces)

### DISCUSSION THREAD

**Discussion by mokazemi** [zen-browser/desktop#3048] [General]
Hi there 👋 

Before anything, thanks for your great effort on making something interesting. It's really stable.

I wanted to suggest something. In my opinion, there is definitely confusion for users about the features used to separate things in the browser (Profiles, containers, workspaces).

**Profiles** are something totally isolated, **containers** are something isolated in the tab level, and **workspaces** are separated in the window level. 

I know each of these might be usable by a group of people, but having all them enabled is really a confusion, even for me that I know the usage of all them. IMO, **the container and workspaces should act as a same thing**, or **workspaces could become something like tab groups.**  Or even **making workspaces and profiles the same thing.** It's better that being something in between.
(I know you're working on tab groups too, but having them too is totally a mess!!)

**Comment by jinbatsu** [comment]
> I wanted to suggest something. In my opinion, there is definitely confusion for users about the features used to separate things in the browser (Profiles, containers, workspaces).

Agree, it would be great to include headline guide or something in the introduction, that will explain the scoupe and usage of that cases (Profiles, containers, workspaces).

> **containers** are something isolated in the tab level
> IMO, **the container and workspaces should act as a same thing**

I usually use containers for testing the same page with different session, like seeing web as loggedin user, other container as guess user, and other as admin user, etc.. That should not act like workspace, because then I should jump to my workspace to see side by side, which is I prefer in the same workspace.
